packages/common/src/services/integrations/jwp/JWPCheckoutService.ts

```diff
@@ -276,13 +283,50 @@ export default class JWPCheckoutService extends CheckoutService {
     }
   };
 
-  getSubscriptionSwitches = undefined;
+  getSubscriptionSwitches: GetSubscriptionSwitches = async (payload) => {
+    const { data } = await InPlayer.Asset.getAssetAccessFees(this.parseOfferId(payload.offerId));
+
+    const subscriptionSwitches = data?.filter((accessFee) => accessFee.item.plan_switch_enabled).map((accessFee) => this.formatOffer(accessFee)) || [];
+
+    return { responseData: subscriptionSwitches, errors: [] };
+  };
 
   getOrder = undefined;
 
-  switchSubscription = undefined;
+  switchSubscription: SwitchSubscription = async (payload) => {
```

Comment:
Can we also remove `changeSubscription` method from subscription service?

Reply:
Yes!